lib/foreman_maintain/config.rb

```diff
@@ -27,6 +25,19 @@ def load_log_configs
       @log_file_size = @options.fetch(:log_file_size, 10_000)
     end
 
+    def load_backup_dir_paths
+      @storage_file = @options.fetch(:storage_file, 'data.yml')
+      @backup_dir = find_dir_path(
+        @options.fetch(:backup_dir, '/var/lib/foreman-maintain')
+      )
+      @db_backup_directories = {}
+      @db_backup_directories = {
+        :candlepin => find_dir_path(
```

Comment:
As I mentioned in the settings, I think we would be ok with having just one backup dir for now, not specific to just candlepin. It also might change as we move the backup/restore scripts.